app/assess/helpers.py

```diff
@@ -129,26 +129,26 @@ def is_flaggable(flag_status: str):
     return flag_status != "Stopped"
 
 
-def is_qa_complete(Flags: List[FlagV2]) -> bool:
+def is_qa_complete(Flags: List[Flag]) -> bool:
     # TODO: Rework on this when QA_COMPLETED is moved to assessment enum type
     flags_list = (
         [
-            (FlagV2.from_dict(flag) if isinstance(flag, dict) else flag)
+            (Flag.from_dict(flag) if isinstance(flag, dict) else flag)
             for flag in Flags
         ]
         if Flags
         else []
     )
     all_latest_status = [flag.latest_status for flag in flags_list]
-    return FlagTypeV2.QA_COMPLETED in all_latest_status or False
+    return FlagType.QA_COMPLETED in all_latest_status or False
```

Comment:
we can remove this function. `QA_COMPLETED` status is not a part of flags status anymore